docs/application/get-started/tv.md

```diff
@@ -1,32 +1,37 @@
+For TV, Tizen supports two primary application types, .NET and Web applications.
 
-The Get Started Tutorial teaches you how to create basic Tizen applications for TV devices.
-
-Tizen supports two primary application types for TV:
-
--   [.NET application](#dotnet-application) is a new way to develop applications for the Tizen operating system using C# language.
-
--   [Web application](#web-application) is a Web site stored on your device and built using Web-native languages, such as HTML5, CSS, and JavaScript. The Web application uses the Tizen Web framework to interact with the native subsystems.
-
-<a name="dotnet-application"> </a>
-## .NET Application
-
-The following guide demonstrates how to create and run a basic Tizen .NET application:
+## TV .NET Applications
+You can create rich applications in C# using the .NET framework benefits that contribute to secure and efficient application development. 
 
+To begin with, you can use the following quick start guide:
 -   [TV .NET application](../dotnet/get-started/tv/first-app.md)
 
-First, ensure that you have installed the latest version of Visual Studio Tools for Tizen. [**Learn more >**](../vstools/install.md)
-
-## Web Application
+Ensure that you have installed the latest version of [Visual Studio Tools for Tizen](https://marketplace.visualstudio.com/items?itemName=tizen.VSToolsforTizen).
+	
+Explore more about the Tizen TV .NET application development:
+-   [Tutorials](../dotnet/tutorials/overview.md)	
+-   [Guides](../dotnet/guides/overview.md)
+-   [API reference ](../dotnet/api/overview.md)
 
-This following guide demonstrates how to create and run a basic Tizen Web application:
+## TV Web Applications
+You can create rich web applications using web-native languages, such as HTML, JavaScript, and CSS that are combined in a package. These applications use the Tizen Web Framework to interact with the native subsystems.
 
+To begin with, you can use the following quick start guides:
 -   [TV Web application](../web/get-started/tv/first-app.md)
+-   [Samsung Smart TV Web application](../web/get-started/tv/first-samsung-tv-app.md)
+
+Ensure that you have installed the latest version of [Tizen Studio](https://developer.tizen.org/development/tizen-studio/download).
 
+Explore more about the Tizen TV Web application development:
+-   [Tutorials](../web/tutorials/overview.md)
+-   [Guides](../web/guides/index.md)
+-   [API reference](../web/api/index.md) 
 
-This following guide demonstrates how to create and run a simple Tizen Web application for **Samsung Smart TV**:
+Further enrichment with Tizen TV applications:
+-   Connect with the [community](https://developer.tizen.org/community/tip-tech).
+-   [Design](../design/tv/design-principles.md) comfortable visual identity for your applications.
+-   [Distribute](../distribute/how-to-distribute/overview.md) through Tizen Store.
```

Comment:
I think it would be better to delete this block.
Linking to Community on TD is not recommended because Community is not active.
Design and distribute guides are not official documents on docs.tizen.org.

Reply:
Sure, I will remove this block.